script.tag-generator/main.py

```diff
@@ -0,0 +1,643 @@
+from bs4 import BeautifulSoup
+import datetime
+import json
+import os
+import re
+import requests
+import simplejson
```

Comment:
drop

Reply:
> 
> 
> drop
replaced simplejson calls with json and dropped the import of simplejson